Sources/TensorFlow/Loss.swift

```diff
@@ -43,6 +49,7 @@ public func l2Loss<Scalar: TensorFlowFloatingPoint>(
 }
 
 /// Returns the mean absolute error between predictions and expectations.
+/// Applies the mean reduction to the l1 loss
```

Comment:
Does this line add significant information that is missing in the line above? If so, this may need to be rephrased so that it flows better with the existing comment. Additionally, a period is needed.

The same can apply to the added comment just below this one.

Reply:
Well, it explains a little bit better the computation used, but maybe it's superfluous?

Reply:
I agree with @BradLarson, and I think maybe we could have something like `Returns the mean absolute error (L1) between...` to make it more clear that `l1Loss<Scalar: TensorFlowFloatingPoint>` is being used and we are taking the mean of it. But even that may not be necessary as L1 is defined to be the absolute error. So I'm personally leaning towards not including this addition. Same for the added comment below for mean L2 loss.